pkg/controller/plan/adapter/ovirt/validator.go

```diff
@@ -119,3 +120,19 @@ func (r *Validator) MaintenanceMode(_ ref.Ref) (ok bool, err error) {
 	ok = true
 	return
 }
+
+// Validate storage map is dynamic for populator
+func (r *Validator) StorageDynamic(storageClasses v1.StorageClassList) bool {
+	if r.plan.Spec.Warm || !r.plan.Provider.Destination.IsHost() {
+		return true
+	}
```

Comment:
this part is not so great because it return 'true' also when the storage is not using dynamic provisioning